src/map/battle.cpp

```diff
@@ -5954,6 +5955,13 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 				skillratio += 150 * skill_lv;
 			RE_LVL_DMOD(100);
 			break;
+		case SHC_CROSS_SLASH:
+			skillratio += -100 + 300 * skill_lv + 5 * sstatus->pow;	// !TODO: check POW ratio
+
+			if( sc != nullptr && sc->getSCE( SC_SHADOW_EXCEED ) )
+				skillratio += 60 * skill_lv;
```

Comment:
```suggestion
			if( sc != nullptr && sc->getSCE( SC_SHADOW_EXCEED ) ) {
				skillratio += 60 * skill_lv;
				skillratio += 2 * sstatus->pow;
			}
```
+2 POW ratio with SC_SHADOW_EXCEED match the damage